README.md

```diff
@@ -180,6 +181,38 @@ Flynt includes several utility functions for creating Advanced Custom Fields opt
 * `Flynt\Utils\Options::getTranslatable` <br> Retrieve a translatable option.
 * `Flynt\Utils\Options::getGlobal` <br> Retrieve a global option.
 
+### Resize Dynamic
+
+Timber provides [a `resize` filter to resize images](https://timber.github.io/docs/reference/timber-imagehelper/#resize). This filter creates all images on the page when it's loaded for the first time. If there are many images on one page then this can lead to a very slow load time, or even a complete timeout.
+
+Flynt solves this with the `resizeDynamic` filter. This filter only generates images when the image is requested, rather than when the page is loaded.
+
+All of the generated images are stored in `uploads/dynamic`. If you want to manually regenarate all of these images you can delete this folder and the next time an image is requested it will be regenerated.
+
+#### WebP Support
+
+`resizeDynamic` creates a WebP file of each image it creates. For this to work, it adds a rewrite rule to `.htaccess` so that Apache will automatically serve the WebP version to all browsers that support it.
+
+#### Troubleshooting
+
+In order for `resizeDynamic` to work, Flynt needs to know the root relative url of the uploads folder. If you have a non-standard WordPress folder structure, or if you use a plugin that manipulates `home_url` (for example, [WPML](https://wpml.org/)) this can cause problems when using `resizeDynamic`.
+
+If your images do not load, try manually setting the relative uploads directory and refreshing the permalink settings in the back-end:
+
+```php
+add_filter('Flynt/TimberDynamicResize/relativeUploadDir', function () {
+    return '/app/uploads'; // Example for Bedrock installs.
+});
+```
+
+#### Disable `resizeDynamic`
+
+To disable `resizeDynamic` completely, add this filter to your theme. Then it will fallback to the regular Timber `resize` functionality.
+
+```php
+add_filter('Flynt/TimberDynamicResize/disable', __return_true);
+```
```

Comment:
@steffenbew @domtra I took a crack at simplifying it, let me know what you think